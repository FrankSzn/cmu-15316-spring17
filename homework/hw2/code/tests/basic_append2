as principal admin password "admin" do      
   set x = []
   set y = []
   append to x with "first element"
   append to y with {field = "data", value = "second element"}
   append to x with y
   return x
***