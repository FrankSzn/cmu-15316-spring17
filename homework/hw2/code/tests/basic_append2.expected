{"status":"SET"}
{"status":"SET"}
{"status":"APPEND"}
{"status":"APPEND"}
{"status":"APPEND"}
{"output":["first element",{"field":"data","value":"second element"}],"status":"RETURNING"}
